napari/components/layerlist.py

```diff
@@ -173,3 +173,9 @@ def select_previous(self, shift=False):
                 self[selected[0] - 1].selected = True
         elif len(self) > 0:
             self[0].selected = True
+
+    def toggle_selected_visibility(self):
+        """Toggle visibility of selected layers"""
+        for i in range(len(self)):
+            if self[i].selected:
+                self[i].visible = not self[i].visible
```

Comment:
```suggestion
        for layer in range(self):
            if layer.selected:
                layer.visible = not layer.visible
```
Minor request to change, but because this class subclasses a list you can actually just iterate right through it, making the code slightly more readable

Reply:
👌 good point!